bin/rule-generator.js

```diff
@@ -0,0 +1,448 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+const figlet = require('figlet');
+const inquirer = require('inquirer');
+const cmd = require('node-command-line');
+const axeFile = path.join(__dirname, '..', 'axe.js');
+const filehound = require('filehound');
+
+const DIRECTORIES = {
+	RULES: path.join(__dirname, '..', '/lib/rules'),
+	TEST_RULES_MATCHES: path.join(__dirname, '..', '/test/rule-matches'),
+	CHECKS: path.join(__dirname, '..', '/lib/checks'),
+	TEST_CHECKS_UNIT: path.join(__dirname, '..', '/test/checks'),
+	TEST_CHECKS_INTEGRATION: path.join(__dirname, '..', '/test/integration/rules')
+};
+
+const showBanner = () => {
+	const COLOR_DQ_HEX = '#0077c8';
+	console.log(chalk.hex(COLOR_DQ_HEX)(figlet.textSync('Axe Rule Generator')));
+};
+
+const QUESTIONS = {
+	GET_RULE_NAME: {
```

Comment:
Not a big deal, but WHY IS NEARLY EVERY VARIABLE IN THIS FILE UPPER-CASED? 😄

Reply:
Meh! I have to break away from this habit. I blame my c# past.